src/index.js

```diff
@@ -150,11 +159,13 @@ class FtpServer extends EventEmitter {
       this.server.close((err) => {
         this.log.info('Server closing...');
         if (err) this.log.error(err, 'Error closing server');
+        this.emit('closing', {error: err});
```

Comment:
This should be above, around L154, to be the most accurate - otherwise, it will just emit when the close has been completed.

Reply:
Hi @matt-forster,
Thanks for reviewing this PR!  Hopefully you enjoyed your holidays...

I originally added the  emitter at this location, because I 'thought' that the closing of the server was not always triggered explicit by calling the 'close' function.  I mean when it is being closed due to some unexpected situation...

I have now moved the emitter to the `close` function.  And the readme page has been updated to remove the `error` parameter from the `close` event handler documentation.
Bart